Tests/ValidationTests.swift

```diff
@@ -221,14 +221,14 @@ final class ContentTypeValidationTestCase: BaseTestCase {
 
         // When
         AF.request(endpoint)
-            .validate(contentType: ["application/octet-stream"])
+            .validate(contentType: ["application/octet-stream", "application/json"])
```

Comment:
A single test with two items is too likely to succeed randomly. Can you add a specific test, or update this test, to use many content types, like the image content types you mentioned? That should ensure the whole list is being alphabetized.

Reply:
I'm not sure why you marked this resolved, you didn't add any tests or make this test more complex.